diff --git a/dist/bundle.js b/dist/bundle.js
index 21fda6b204e7fc2cd3e28bafd644ea956fc7d4d6..3277841976d09bbc34949ee0dc2dc92b2fcaa671 100644
--- a/dist/bundle.js
+++ b/dist/bundle.js
@@ -4958,6 +4958,8 @@
                     processFixedComment(comment, this.collision[comment.loc], this.timeline);
                 }
             }
+            this.comments.push(...comments);
+            this.processedCommentIndex = this.comments.length - 1;
         }
         drawCanvas(vpos, forceRendering = false, cursor) {
             const vposInt = Math.floor(vpos);
